content/en/cloud_cost_management/planning/commitment_programs.md

```diff
@@ -0,0 +1,170 @@
+
+With this feature, you can:
+
+- Track and address unused or underused commitments.
+- Target high {{< tooltip text="on-demand" tooltip="Cloud resources billed at standard rates, without any commitment or discount program." >}} spend with additional commitments.
+- Monitor expirations and plan timely renewals.
+- Reduce waste by right-sizing or reallocating commitments.
+
+{{< img src="cloud_cost/planning/planning-commitments-overview.png" alt="Dashboard showing filters, KPIs (Effective Savings Rate, Absolute Savings, Coverage, Utilization), a cost-over-time bar chart, and a table of on-demand hot-spots by region, instance family, and database engine." style="width:100%;" >}}
+
+This page describes the information available on the [Commitment Programs page][1] so you can better understand how your commitments are being utilized and take the necessary action.
+
+## Getting started
+
+Navigate the Commitment Programs page and take action.
+
+1. Access the Commitment Programs page in Cloud Cost Management by navigating to: [**Cloud Cost > Planning > Commitment Programs**][1].
+2. **Review the KPIs** in the [commitments overview](#commitments-overview) section.
+3. **Use filters** to focus on specific accounts, regions, or services.
+4. **Explore the [costs overview](#costs-overview)** to analyze utilization, coverage, and expiring commitments.
+5. **Review expiration dates and renewal recommendations in the [Commitments Explorer](#commitments-explorer)**.
+6. **Take action**. Based on the insights from the Commitment Programs page, you can:
+
+   - Adjust workloads to make better use of your existing commitments and avoid extra on-demand charges.
+   - Update commitments by buying or changing them based on your usage data.
+   - Plan renewals by renewing or retiring commitments before they expire.
+   - Optimize spend by using Datadog's tips to save more and reduce waste.
+
+## Commitments overview
+
+The commitments overview section is where you can review the following Key Performance Indicators (KPI) for your cloud providers and services.
+
+{{< img src="cloud_cost/planning/commitments-overview.png" alt="Dashboard summarizing cloud commitment KPIs, providing a quick overview of savings performance and highlights areas needing attention." style="width:100%;" >}}
+
+- **Effective Savings Rate (ESR):** The percentage of cost savings achieved by your discount programs compared to on-demand prices, factoring in both utilized and underutilized commitments.
+
+  _Example: Coverage is 90% but ESR is 65%—you may have underutilized commitments dragging down your savings._
```

Comment:
- The ESR is capped by the maximum discount rate achievable with the program you're looking at - so it won't get any higher than 64% for RDS (3yr, all upfront RI) or 62% for EC2 (3yr, all upfront, standard RI)

- Since ESR is only about costs covered by Reserved Instances, I'm not sure it makes sense to mention coverage in the example?